content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,827 @@
+
+Kanban view
 ===========
-Recruitment
-===========
+
+To access the kanban view for a job position, navigate to the main :menuselection:`Recruitment app`
+dashboard, which is the default view when opening the application. All job positions appear on the
+main dashboard. Click the :guilabel:`New Applications` smart button on a job position card to
+navigate to the kanban view for all the applicants for that particular job position.
+
+.. image:: recruitment/new-applicants-button.png
+   :align: center
+   :alt: Main dashboard view of job position card, showing new applications button.
+
+Inside the job application, the kanban stages appear, with all the applicants populated in their
+respective columns, indicating what stage they are currently in. There are five default stages in
+Odoo:
+
+- :ref:`Initial Qualification <initial>`
+- :ref:`First Interview <interview>`
+- :ref:`Second Interview <second>`
+- :ref:`Contract Proposal <contract>`
+- :ref:`Contract Signed <signed>`
+
+The last column, :guilabel:`Contract Signed`, is folded by default, meaning that the column appears
+in gray and the applicants in it are hidden from view. To expand the folded stage and view the
+applicant cards for that column, click anywhere on the thin gray column that says the stage name and
+the column expands, revealing the applicants.
+
+.. image:: recruitment/stages.png
+   :align: center
+   :alt: Expand a folded column by clicking on it in the kanban view.
+
+Each stage has a color-coded bar beneath the stage name, giving information about the applicant's
+status in the stage. The colors are:
+
+- :guilabel:`Green`: There are applicants in the stage with an activity that is either already
+  scheduled or needs to be scheduled in the future (such as a phone call or interview), according to
+  the parameters for that particular stage.
+- :guilabel:`Yellow`: There are applicants in the stage with an activity either scheduled for today
+  or due to be scheduled today.
+- :guilabel:`Red`: There are applicants in the stage with overdue activities which need to be
+  scheduled.
+- :guilabel:`Gray`: There are either no activities currently scheduled and no future activities that
+  are required to be scheduled, or there are no applicants currently in the stage.
+
+.. _stages:
+
+Customize stages
+================
+
+Stages can be modified, added, or deleted to best meet the needs of the particular hiring steps of
+a business.
+
+New stage
+---------
+
+To create a new stage, click on :guilabel:`+ Add a Column` and a new column appears. Enter the title
+for the new stage in the :guilabel:`Column title` field, then click :guilabel:`Add`. The new column
+appears, and another new stage is available to create. If no new stages are needed, click anywhere
+on the screen to exit the new column creation.
+
+.. image:: recruitment/add-column.png
+   :align: center
+   :alt: The gear icon that appears when a column name is moused over, and the menu it displays when
```

Comment:
This alt text does not seem to apply to this image in context, I suppose it is for the next image that is missing an alt tag description 😄